readconfig.go

```diff
@@ -52,4 +76,6 @@ type QueueWorkerConfig struct {
 	FunctionSuffix string
 	DebugPrintBody bool
 	WriteDebug     bool
+	maxInflight    int
```

Comment:
I would like these to match the style of the other variables.

Reply:
@alexellis ok, I will modify